examples/tests/invalid/access_private_apis.test.w

```diff
@@ -0,0 +1,18 @@
+bring "./subdir2" as subdir2;
+
+new subdir2.MyPrivateClass();
```

Comment:
Let's make sure to fast-follow with [`internal`](https://github.com/winglang/wing/issues/4156) because there's definitely a use case for sharing types within a library but not expose them outside of it.